### ISSUE TITLE
Collection info page should list specimens

### ISSUE THREAD

**Issue by vjpsyverson** [paleobiodb/PBDB-Wing#44]
Right now there is no way to get to the list of specimens related to a collection from that collection's page, only from the specimen search form. This is pretty annoying.

**Comment by markuhen** [comment]
I don't know what you mean. Do you mean occurrences? The collection page does indeed list those. Please try to be more specific. See below.

![melvin](https://cloud.githubusercontent.com/assets/19557452/25246224/fbb0c5f0-25d4-11e7-92a4-f916e55243dd.jpeg)

**Comment by vjpsyverson** [comment]
No, I mean the list of specimens and measurements associated with the collection. You can presently get the page I'm talking about by pasting the collection number into the specimen search form under "Add/Edit measurements", but that's the only way.

**Comment by markuhen** [comment]
Indeed, this data has only been poorly tacked on and integrated with the rest of the system.
